lib/eu_central_bank.rb

```diff
@@ -209,13 +209,19 @@ def update_parsed_historical_rates(doc)
     @historical_last_updated = Time.now
   end
 
+
+  #new method for round
+  def exchange_with_round(from, to_currency, round = 2, date = nil)
+    exchange_with(from, to_currency, date).round(round)
```

Comment:
This use of the `.round` is incorrect since its footprint is different from the `Numeric#round` (unfortunately and confusingly, but that's the case) — https://github.com/RubyMoney/money/blob/master/lib/money/money.rb#L611